Web/src/components/games/BalloonShake/GamePrep.tsx

```diff
@@ -30,23 +30,23 @@ const GamePrep: FC<IProps> = props => {
   return (
     <Container>
       {permission === MotionPermission.NOT_SET && !showPermissionRequest
-          && (
-            <h3>
-              Checking if you can play the game...
-            </h3>
-          )}
+        && (
+          <h3>
+            Checking if you can play the game...
+          </h3>
+        )}
       {permission === MotionPermission.NOT_SET && showPermissionRequest
-          && (
-            <Button onClick={() => requestPermissionCallback()} type="button">
-              Set Permission
-            </Button>
-          )}
+        && (
+          <Button onClick={() => requestPermissionCallback()} type="button">
+            Set Permission
+          </Button>
+        )}
       {permission === MotionPermission.GRANTED
-          && (
```

Comment:
Seems like we're using a different formatter somehow cos you keep on formatting it this way but my VSCode kept on reformatting it in another way.

Just wondering, what formatter do you use? Cos I think eslint doesn't really have much opinion on continuation indentation

Reply:
uhh i normally run `yarn lint --fix`